DSA-Campaign-Uplift-Estimation/src/main/webapp/script.js

```diff
@@ -11,3 +11,21 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
+// Checks user's login status and sets login link accordingly.
+function getLogsStatus() {
+
+    fetch('/userapi').then(response => response.json()).then(logStatus => {
+        const link = document.getElementById("login-link");
+
+        if(logStatus.isLoggedIn) {
+
+            console.log('User is Logged In');
+            link.href = '../Home/home.html';
+        }
+        else {
+            console.log('User is not Logged In');
+            link.href = logStatus.Url;
+        }
+    });
+}
```

Comment:
This code is still repeated in DSA-Campaign-Uplift-Estimation/src/main/webapp/Resources/NavBar/navbar_script.js and DSA-Campaign-Uplift-Estimation/src/main/webapp/script.js. Can we consolidate the two?

Reply:
Tried and did it. Done.